package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "warehouse-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@trendmicro/react-sidenav": "^0.4.5",
+    "bootstrap": "^4.3.1",
+    "react": "^16.11.0",
+    "react-cookie": "^4.0.1",
+    "react-dom": "^16.10.2",
+    "react-responsive-modal": "^4.0.1",
+    "react-router-dom": "^5.0.1",
+    "reactstrap": "^8.1.1"
+  },
+  "scripts": {
+    "start": "react-scripts start",
```

Comment:
react-scripts cannot be used for the PRODUCTION build.
https://create-react-app.dev/docs/available-scripts
```
npm start
Runs the app in the development mode. Open http://localhost:3000 to view it in the browser.
```
Please read about any module/package before ussing them